modules/ROOT/pages/custom-policy-getting-started.adoc

```diff
@@ -13,18 +13,13 @@ The current workflow to get a working policy for Mule 4 that can be applied in A
 . Upload the resulting policy assets to Exchange.
 . Apply the policy to any API through API Manager.
 
-
 == Setting up a project with the archetype
 
-The first step of developing a custom policy consists in setting up a project with the files needed for creating a policy.
-We are going to use Maven in order to do that and also for packaging it into a deployable JAR.
-
-The easiest way to do this is through the use of a maven archetype that will create necessary files for us.
+The first step to develop a custom policy consists in setting up a project with the required files. +
+The easiest way to bring all your required files is by using the maven archetype. This archetype creates all the necessary files for you.
+You also need to use Maven package your custom policy into a deployable JAR.
```

Comment:
Not super clear to me what the relationship of line 19 & 20 is. I think we mean "Then, use Maven to package your custom policy...."?

Reply:
Also, "gather" might be clearer than "bring". Bring implies taking something somewhere more than taking a bunch of things and putting them in one place.